functions.test.js

```diff
@@ -0,0 +1,20 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEquals = require('.\\functions')
+
+
+describe('Functions', () => {
+    describe('strictEquals', () => {
+        it('checks for equality and returns a boolean', () => {
+
+            const values = ('a', 'a')
+
+            const val = strictEquals(values)
```

Comment:
Review functions and parameters. You're missing some very fundamental concepts right now and you are going to be lost going forward without nailing them down.